src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -917,6 +917,62 @@ public EventAccessorInfo(MethodInfo interfaceAddMethod, MethodInfo interfaceRemo
                 {
                     InterfaceAddMethod = interfaceAddMethod;
                     InterfaceRemoveMethod = interfaceRemoveMethod;
+
+            private sealed class MethodInfoEqualityComparer : EqualityComparer<MethodInfo>
+            {
+                public static readonly MethodInfoEqualityComparer Instance = new MethodInfoEqualityComparer();
+
+                private MethodInfoEqualityComparer() { }
+
+                public override bool Equals(MethodInfo left, MethodInfo right)
```

Comment:
This may seem overzealous but I like to seal the override methods too. 

- Makes it easier for reader to know that the method is sealed without scrolling up to the type definition.

- If the type is ever unsealed and derived from, this practice ensures that the methods will only be unsealed on an as-needed basis instead of them all becoming unsealed at once.